src/PlusVolumeReconstruction/vtkPlusPasteSliceIntoVolumeHelperCommon.h

```diff
@@ -344,6 +344,7 @@ static int vtkTrilinearInterpolation(F *point,
           break;
         }
         inPtrTmp++;
+        importancePtr++;

```

Comment:
Are you sure you want to increment importancePtr for each scalar component?
As I saw, importance image is always a single-component image, while input and output may have multiple components.